src/amp-story-player-impl.js

```diff
@@ -45,6 +46,9 @@ const IframePosition = {
   NEXT: 1,
 };
 
+/** @const @type {!Array<string>} */
+const SUPPORTED_CACHES = ['cdn.ampproject.org', 'www.bing-amp.com'];
```

Comment:
These are available in `caches.json`. Would it be possible to pull these values from this json file so it stays consistent as they're added/removed?

Reply:
+1